### PR TITLE
Daccess 346

### PR DESCRIPTION
**Pull request by mhk33**
This PR adds separate headings for "Looking for more?" and "Limit your search" in the facet column of the results. Previously all of the links were under "Limit your search," but that doesn't really make sense for the external links where you might be expanding your search beyond the catalog.

### CODE REVIEW

File: blacklight-cornell/app/assets/stylesheets/cornell/_facets.scss [hunk 1]
```diff
@@ -39,7 +39,9 @@
 	}
 }
 .facets-heading {
-	font-size: 1.25rem;
+	font-size: 1.25rem
```
Comment (chrisrlc) [inline blacklight-cornell/app/assets/stylesheets/cornell/_facets.scss:42]: should this have a semicolon at end? also can the 2 lines of commented out css be cleaned out, or is that being saved for later?

File: blacklight-cornell/app/assets/stylesheets/singlesearch.scss [hunk 2]
```diff
@@ -162,7 +162,8 @@ div.view-all {
     font-size: 1rem;
   }
   i {
-    color: $concrete;
+    // color: $concrete;
```
Comment (chrisrlc) [inline blacklight-cornell/app/assets/stylesheets/singlesearch.scss:165]: same comment as above, re: cleaning up commented out css.

File: blacklight-cornell/app/assets/stylesheets/cornell/_facets.scss (hunk 1 above)
Comment (mhk33) [reply blacklight-cornell/app/assets/stylesheets/cornell/_facets.scss:42]: yes, it should, for consistency!

File: blacklight-cornell/app/assets/stylesheets/singlesearch.scss (hunk 2 above)
Comment (mhk33) [reply blacklight-cornell/app/assets/stylesheets/singlesearch.scss:165]: yes... should clean up my WIP!

**Review by sarah-cul — APPROVED** [review]
looking forward to seeing this live!

**Review by chrisrlc — APPROVED** [review]
Looks good!

**Pull request merged**